pallets/identity/rpc/src/lib.rs

```diff
@@ -53,16 +60,28 @@ pub enum Error {
     RuntimeError,
 }
 
-impl<C, Block, IdentityId, Ticker> IdentityApi<<Block as BlockT>::Hash, IdentityId, Ticker>
+impl From<Error> for i64 {
```

Comment:
Enum can be cast as integers (i32): see [Enums C-like](https://doc.rust-lang.org/rust-by-example/custom_types/enum/c_like.html)